packages/@storefront/core/tsconfig.json

```diff
@@ -7,7 +7,9 @@
       "DOM"
     ],
     "types": [
-      "node"
+      "node",
```

Comment:
@jmbtutor should we do this as a separate PR?

Reply:
Yes.

Reply:
👍 Jesse can you make a separate PR for the tsconfig stuff? I see the ticket is in "ready for review" but I don't see a PR for it

Reply:
@Vic-Dev Yup, submitted as #10.